### DISCUSSION TITLE
Is delay supposed to invert the signal?

### DISCUSSION THREAD

**Discussion by treefrogman** [maximecb/noisecraft#92] [General]
![image](https://user-images.githubusercontent.com/985484/169676679-4ea6bcdb-1774-4a31-8d02-c05a733457c0.png)

**Comment by treefrogman** [comment]
I can't seem to reproduce this again, after doing so twice...

**Reply by maximecb** [reply]
Weird! I tried reproducing it here and also couldn't seem to manage with a sine or a square wave. Hmm.

**Comment by treefrogman** [comment]
It happened again. This time I tried stopping and starting, and that fixed it. So it appears to be a glitch that develops from live rewiring.

**Comment by maximecb** [comment]
Okay hmm well. Could you please share the project exhibiting the problem?

Would you also be willing to help me find the cause?

**Reply by treefrogman** [reply]
I didn't save a copy of the project at that point, but I will try to reproduce it again.
